src/main/java/bio/terra/service/filedata/google/gcs/GcsIO.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.service.filedata.google.gcs;
+
+import bio.terra.service.resourcemanagement.exception.GoogleResourceException;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobInfo;
+import com.google.cloud.storage.Storage;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+
+public class GcsIO {
+
+  public static String getBlobContents(Storage storage, String projectId, BlobInfo blobInfo) {
+    var blob = storage.get(blobInfo.getBlobId(), Storage.BlobGetOption.userProject(projectId));
```

Comment:
This classes all accept `storage` as the first argument. If we already have a class wrapper around `storage` (`GcsPdao`) and the only uses of these APIs are from that object, then it seems like they could be non-static methods on that class instead. I think the only reason to refactor them out is if they're used by a different `Storage` object outside of `GcsPdao`.

Reply:
`GcsPdao` doesn't wrap `storage`. Methods within `GcsPdao` get `storage` for datasets, projects, etc, then do stuff with that storage. So, even if these were migrated into `GcsPdao`, we'd still have to be passing around `storage`.